src/core/Banner/index.tsx

```diff
@@ -50,6 +50,7 @@ const Banner = forwardRef<HTMLDivElement, BannerProps>(function Banner(
       </Centered>
       {dismissible && (
         <StyledButtonIcon
+          aria-label="Close"
```

Comment:
I added a lot of aria-labels to icon buttons. Let me know what you think.

Theoretically the MUI icons should take a `titleAccess` prop which is equivalent, but I couldn't actually get that to work right.

Reply:
Could also think about enforcing label in the prop types for icon buttons.